cms/templates/js/xblock-outline.underscore

```diff
@@ -1,7 +1,7 @@
 <% if (parentInfo) { %>
 <li class="outline-item outline-item-<%= xblockType %> <%= includesChildren ? 'is-collapsible' : '' %> is-draggable <%= isCollapsed ? 'is-collapsed' : '' %>"
     data-parent="<%= parentInfo.get('id') %>" data-locator="<%= xblockInfo.get('id') %>">
-    <span class="draggable-drop-indicator draggable-drop-indicator-before"><i class="icon fa fa-caret-right"></i></span>
+    <span class="draggable-drop-indicator draggable-drop-indicator-before"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
Now these drag and drop indicators ARE used. And again, I'm wondering if they need sr text. @clrux can you please look? This is for drag and drop of sections/subsections on the course overview page in Studio.